domain/src/main/java/me/androidbox/domain/alarm_manager/AlarmItem.kt

```diff
@@ -2,12 +2,7 @@ package me.androidbox.domain.alarm_manager
 
 import java.time.ZonedDateTime
 
-
 data class AlarmItem(
```

Comment:
What´s the use of this item?

Reply:
@MKiperszmid I was following @philipplackner tutorial on how to use Alarm Manager, as the time I have never used Alarm Manager. But I guess I can remove this if its not needed.